controller.rst

```diff
@@ -153,11 +139,18 @@ Add the ``use`` statement atop the ``Controller`` class and then modify
         // ...
     }
 
-Helper methods are just shortcuts to using core Symfony functionality
-that's available to you with or without the use of the base
-controller classes. A great way to see the core functionality in
-action is to look in the
-:class:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller` class.
+That's it! You now have access to methods like :ref:`$this->render() <controller-rendering-templates>`
+and many others that you'll learn about next.
+
+.. tip::
+
+    You can extend either ``Controller`` or ``AbstractController``. The difference
+    is that when you extend ``AbstractController``, you can't access services directly
+    via ``$this->get()`` or ``$this->container->get()``. This forces you to write
+    more robust code to access services, but if you're not use, use ``Controller``.
```

Comment:
I would reverse this sentence: `but if you're use to fetch dependencies using the container, use Controller``, wdyt?